GettingStarted/ibmdbpy_GettingStarted_3-geo_guide.ipynb

```diff
@@ -0,0 +1,4152 @@
+      "\n",
+      "CREATE TABLE sample_geometries (id INTEGER, geometry db2gse.ST_Geometry) organize by row;\n",
+      "\n",
+      "INSERT INTO sample_geometries(id, geometry) VALUES (1, db2gse.ST_Geometry('point(1 2)', 1) );\n",
+      "\n",
+      "INSERT INTO sample_geometries(id, geometry) VALUES(2, db2gse.ST_Geometry ('polygon ((0 0, 0 4, 5 4, 5 0, 0 0))', 1) );\n",
+      "\n",
+      "INSERT INTO sample_geometries(id, geometry) VALUES(3, db2gse.ST_Geometry ('point empty', 1) );\n",
+      "\n",
+      "INSERT INTO sample_geometries(id, geometry) VALUES(4, db2gse.ST_Geometry ('multipolygon empty', 1) );  \n",
+      "\n",
+      "INSERT INTO sample_geometries(id, geometry) VALUES(5, db2gse.ST_Geometry('linestring(33 2, 34 3, 35 6)', 1) );\n",
+      "\n",
+      "INSERT INTO sample_geometries(id, geometry) VALUES(6, db2gse.ST_Geometry('polygon((3 3, 4 6, 5 3, 3 3))', 1));\n",
+      "\n",
+      "INSERT INTO sample_geometries(id, geometry) VALUES(7, db2gse.ST_Geometry('<gml:Point srsName=\";EPSG:4269\";><gml:coord><gml:X>50</gml:X><gml:Y>60</gml:Y></gml:coord></gml:Point>', 1)); \n",
+      "\n",
+      "CREATE TABLE sample_points(id INT GENERATED BY DEFAULT AS IDENTITY NOT NULL, loc db2gse.ST_POINT NOT NULL, PRIMARY KEY (id)) ORGANIZE BY ROW;\n",
+      "\n",
+      "INSERT INTO SAMPLE_POINTS(loc) VALUES (db2gse.ST_Point(14, 58, CAST(NULL AS DOUBLE), 1));\n",
+      "\n",
+      "INSERT INTO SAMPLE_POINTS(loc) VALUES (db2gse.ST_Point(12, 35, 12, 1));\n",
+      "\n",
+      "INSERT INTO SAMPLE_POINTS(loc) VALUES (db2gse.ST_Point(12, 66, 43, 45, 1));\n",
+      "\n",
+      "INSERT INTO SAMPLE_POINTS(loc) VALUES (db2gse.ST_Point(14, 58, CAST(NULL AS DOUBLE), 4, 1));\n",
+      "\n",
+      "INSERT INTO SAMPLE_POINTS(loc) VALUES (db2gse.ST_Point(12, 35, 12, CAST(NULL AS DOUBLE),1));\n",
+      "\n",
+      "INSERT INTO SAMPLE_POINTS(loc) VALUES (db2gse.ST_Point(17, 65, 32, 1));\n",
+      "\n",
+      "INSERT INTO SAMPLE_POINTS(loc) VALUES (db2gse.ST_Point(17, 65, 34, 3, 1));                             \n",
+      "\n",
+      "CREATE TABLE sample_mlines (id INTEGER, geometry db2gse.ST_MultiLineString) ORGANIZE BY ROW;\n",
+      "\n",
+      "INSERT INTO sample_mlines VALUES (1110, db2gse.ST_MultiLineString ('multilinestring ( (33 2, 34 3, 35 6), (28 4, 29 5, 31 8, 43 12), (39 3, 37 4, 36 7) )', 1) );                             \n",
+      "\n",
+      "CREATE TABLE sample_lines(id SMALLINT, geometry db2gse.ST_LineString) organize by row; \n",
+      "\n",
+      "INSERT INTO sample_lines(id, geometry) VALUES (1110, db2gse.ST_LineString('linestring(850 250, 850 850)', 1) );\n",
+      "\n",
+      "INSERT INTO sample_lines(id, geometry) VALUES (1111, db2gse.ST_LineString('<gml:LineString srsName=\";EPSG:4269\";><gml:coord><gml:X>90</gml:X><gml:Y>90</gml:Y></gml:coord><gml:coord><gml:X>100</gml:X><gml:Y>100</gml:Y></gml:coord></gml:LineString>', 1) );  \n"
+     ]
+    }
+   ],
+   "source": [
+    "f=open(\"sql_script\", \"r\")\n",
```

Comment:
Just like an IdaDatabase, you need to close `f` as well. You can use [context managers](https://docs.python.org/3/reference/compound_stmts.html#the-with-statement) to make that happen automatically:

```python
with open(...) as f:
    for line in f:
        [...]
```

Reply:
modified accordingly